signer/core/signed_data.go

```diff
@@ -216,57 +211,6 @@ func (api *SignerAPI) SignTypedData(ctx context.Context, addr common.MixedcaseAd
 	return signature, nil
 }
 
-func parseInteger(encType string, encValue interface{}) (*big.Int, error) {
```

Comment:
Why was this removed? It's still in upstream master

Reply:
because in our codebase this was moved to `shared/signer/signed_data.go`. During the merge it was accidentally duplicated